cmd/minikube/cmd/root.go

```diff
@@ -74,6 +79,7 @@ func Execute() {
 
 func init() {
 	RootCmd.PersistentFlags().BoolVarP(&showLibmachineLogs, "show-libmachine-logs", "", false, "Whether or not to show logs from libmachine.")
+	RootCmd.PersistentFlags().BoolVarP(&libmachineDebug, "debug", "", false, "[Enable libmachine debug mode]")
```

Comment:
What do you think about just defaulting this to true? show-libmachine-logs is already "debug"-y. We probably dont' need two levels.